packages/types/package.json

```
@@ -36,12 +36,12 @@
     "type-check": "tsc"
   },
   "peerDependencies": {
-    "@trpc/server": "^10"
+    "@trpc/server": "11.0.0-rc.593"
```

No need to be strict about peer dependencies.
```suggestion
    "@trpc/server": "^11"
```